### ISSUE TITLE
Profile.get_followees() ordering

### ISSUE THREAD

**Issue by SelfAdjointOperator** [instaloader/instaloader#1592]
**Provide us a use case of the feature**

If I'm not mistaken the feature on Instagram to sort one's own following (followees) list by
- `Default`
- `Date followed: Latest`
- `Date followed: Earliest`
from within the following list in the app is a new(ish) feature. It'd be nice to have the iteration order of `Profile.get_followees()` as any of these if desired.

**Describe the solution you'd like**
Perhaps the signature of
`Profile.get_followees(self)`
could be adapted to
`Profile.get_followees(self, iteration_order: str = "default")`
where `iteration_order` could be a string such as `"default"`, `"earliest_to_latest"`, or `"latest_to_earliest"` (or a corresponding enum)

**Describe alternatives you've considered**
I'm not sure if the timestamp of when one's account begun following an account can be accessed from Instagram's API? Or at least it is specific only to profiles one owns. If this were available then one could sort a list of profiles by these timestamps, however that requires iterating through all followers at once to create the list.

**If the feature request is accepted, would you be willing to submit a pull request?**
I am not too familiar with Instagram's API but yes if the instructions were clear. I've contributed to the documentation here before lol.

![Screenshot_20220619-010838 scale](https://user-images.githubusercontent.com/66702639/174460797-f5940a16-1075-4f18-8510-241fca58474c.jpg)

**Comment by aandergr** [comment]
Yes, this looks like a useful feature to me and I will be happy to review a pull request for it!
